Mage/src/main/java/mage/abilities/ActivatedAbilityImpl.java

```diff
@@ -180,13 +181,13 @@ public ActivationStatus canActivate(UUID playerId, Game game) {
         // timing check
         //20091005 - 602.5d/602.5e
         boolean asInstant;
-        ApprovingObject approvingObject = game.getContinuousEffects()
+        Set<ApprovingObject> approvingObjects = game.getContinuousEffects()
```

Comment:
What about that comment?

![shot_230905_211757](https://github.com/magefree/mage/assets/8344157/cb15279d-64ad-466a-b1bc-c0095d5fdcbb)

Reply:
I have no idea and did not touch the `activatorId` current logic, this will require another investigation.